README.md

```diff
@@ -1 +1,95 @@
-# React-bookstore
\ No newline at end of file
+# app-books
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This a personal project made using react and webpack
+
+# WORKFLOW SCREENSHOT
+
+![]
+
```

Comment:
- [ ] The `screenshot`, `live link`, and MIT link in the README file are broken. I would highly recommend fixing those sections to make README professional.